### PR TITLE
removed database context from codebase

### PR DESCRIPTION
**Pull request by A1Liu**
Removed database context from codebase. Wasn't the right abstraction.

### CODE REVIEW

**Comment by NicholasLYang** [comment]
Getting a compile error, I think we need to bump rocket's version

File: src/graphql/mutation.rs [hunk 1]
```diff
@@ -32,13 +32,16 @@ graphql_object!(MutationRoot: RequestContext as "Mutation" |&self| {
         project: ProjectRequest,
         token: String
     ) -> FieldResult<Tokenized<Project>> {
+        use crate::resolvers::{user, project};
```
Comment (NicholasLYang) [inline src/graphql/mutation.rs]: I'm not a huge fan of embedding imports in code. Why not import at top?

Comment (A1Liu) [reply src/graphql/mutation.rs]: Idk, tbh I kinda get uneasy about having a bunch of names in the file scope, but I don't have a good reason for or against, it's just personal preference and conditioning from writing in C I guess.

**Comment by A1Liu** [comment]
Ran `cargo update`, should compile correctly now (at least it works on my machine with rust nightly 1.37.0 and the current `Cargo.lock` configuration)

File: src/graphql/mutation.rs (hunk 1 above)
Comment (NicholasLYang) [reply src/graphql/mutation.rs]: Let's stick to imports on top. I like being able to see what's being imported in one place. It's completely subjective, but we might as well stick to one system.

Comment (A1Liu) [reply src/graphql/mutation.rs]: Ok, fixed!

**Pull request merged**